MapboxSearchUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'MapboxSearchUI'
-  s.version          = '2.8.0-rc.2'
+  s.version          = '2.8.0-rc.3'
```

Comment:
@chizhavko it looks like this version was not updated and [MapboxSearchUI](https://github.com/CocoaPods/Specs/tree/master/Specs/8/a/2/MapboxSearchUI) was not released to cocoapods